Automattic/simplenote-android
# Update Lock Icon

### Fix
Update the lock icon to the new Simplecon, https://github.com/Automattic/Simplecons/pull/8.  The lock icon is shown on the **_Style_** screen when an account is **_Free_**.  See the screenshots below for illustration.

![update_lock_icon](https://user-images.githubusercontent.com/3827611/100154318-9e279900-2e62-11eb-9777-a739dceb2225.png)

### Test
Since the backend for **_Free_**/**_Premium_** accounts is not implemented yet, all accounts are considered **_Premium_** in order to fully develop and test **_Premium_** features.  In order to see the locked styles with the lock icon, update the [`PrefUtils.isPremium(Context)` method](https://github.com/Automattic/simplenote-android/blob/a49d9a6c78d770c188834299d7cda3e719a35b13/Simplenote/src/main/java/com/automattic/simplenote/utils/PrefUtils.java#L165) to return `false`, which will set the account as **_Free_**.
1. Open navigation drawer.
2. Tap **_Settings_** item in drawer.
3. Tap **_Style_** setting under **_Appearance_** section.
4. Notice **_Default_** style is unlocked and selected.
5. Notice other styles are locked with new lock icon.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

#### Note
Contact me for an installable build.

### Release
These changes do not require release notes.  `RELEASE-NOTES.txt` will be updated once the styles feature is complete.
